test/e2e/generator/generate.go

```diff
@@ -174,7 +183,8 @@ func generateTestnet(r *rand.Rand, opt map[string]interface{}) (e2e.Manifest, er
 // generating invalid configurations. We do not set Seeds or PersistentPeers
 // here, since we need to know the overall network topology and startup
 // sequencing.
-func generateNode(r *rand.Rand, mode e2e.Mode, startAt int64, forceArchive bool) *e2e.ManifestNode {
+func generateNode(
+	r *rand.Rand, mode e2e.Mode, startAt int64, forceArchive bool, initialHeight int64) *e2e.ManifestNode {
```

Comment:
Instead of adding `initialHeight`, lets change it to pass `manifest.InitialHeight` instead of `0` when starting at the initial height, and use something like `startAt + 5` or something for misbehavior. Basing it on `InitialHeight` won't work for large networks with e.g. 15 nodes, where the misbehaving node may not even be starting until height 30, yet we schedule the misbehavior for height 20.